crates/ruff/src/rules/flake8_type_checking/snapshots/ruff__rules__flake8_type_checking__tests__runtime-import-in-type-checking-block_TCH004_5.py.snap

```diff
@@ -11,11 +11,11 @@ TCH004_5.py:4:24: TCH004 [*] Move import `typing.List` out of type-checking bloc
 
 ℹ Suggested fix
 1 1 | from typing import TYPE_CHECKING
-  2 |+from typing import List
+  2 |+from typing import List, Sequence, Set
 2 3 | 
 3 4 | if TYPE_CHECKING:
 4   |-    from typing import List, Sequence, Set
-  5 |+    from typing import Sequence, Set
+  5 |+    pass
```

Comment:
The is an unnecessary pass state; i'd expected it to be cleaned up by another rule but it seems there is no rule currently that detects extra `pass` statements? not sure if this is actionable for this PR tough

Reply:
We do have a rule that detects empty type-checking blocks, so this would be picked up from that at least.